misc/alert_rules.json

```diff
@@ -30,5 +30,13 @@
   {
     "rule": "%ports.ifOperStatus = \"down\" && %ports.ifOperStatus_prev = \"up\" && %macros.device_up = \"1\"",
     "name": "Port status change from up to down"
+  },
+  {
+    "rule": "%ports.ifOutErrors_rate >= \"100\" || %ports.ifInErrors_rate >= \"100\"",
+    "name": "Interface Errors Rate"
+  },
+  {
+    "rule": "%eventlog.type = \"discovery\" && %eventlog.message ~ \"autodiscovered\" && %eventlog.datetime >= %macros.past_60m",
```

Comment:
Should be: `\"@autodiscovered@\"`

The name should also be Device discovered within the last 60 minutes.